packages-exp/auth-exp/src/core/user/token_manager.ts

```diff
@@ -100,21 +98,25 @@ export class StsTokenManager {
 
     const manager = new StsTokenManager();
     if (refreshToken) {
-      assert(typeof refreshToken === 'string', AuthErrorCode.INTERNAL_ERROR, {
+      _assert(typeof refreshToken === 'string', AuthErrorCode.INTERNAL_ERROR, {
         appName
       });
       manager.refreshToken = refreshToken;
     }
     if (accessToken) {
-      assert(typeof accessToken === 'string', AuthErrorCode.INTERNAL_ERROR, {
+      _assert(typeof accessToken === 'string', AuthErrorCode.INTERNAL_ERROR, {
         appName
       });
       manager.accessToken = accessToken;
     }
     if (expirationTime) {
-      assert(typeof expirationTime === 'number', AuthErrorCode.INTERNAL_ERROR, {
-        appName
-      });
+      _assert(
```

Comment:
shouldn't this be _assert(assert, auth, code)?

Reply:
We don't have an auth object in the static context

Reply:
We don't have an auth instance here